bin/sozo/src/commands/build.rs

```diff
@@ -22,11 +25,14 @@ pub struct BuildArgs {
     #[arg(long)]
     #[arg(help = "Output directory.", default_value = "bindings")]
     pub bindings_output: String,
+
+    #[arg(long, help = "Display statistics about the compiled contracts")]
+    pub stats: bool,
 }
 
 impl BuildArgs {
     pub fn run(self, config: &Config) -> Result<()> {
-        let compile_info = compile_workspace(
+        let compile_info: dojo_lang::scarb_internal::CompileInfo = compile_workspace(
```

Comment:
The compiler wasn't able to infer the type here? I join @kariy in the way that we should let the compiler infer when it's possible.

Reply:
The compiler was able to infer it, my bad here.